### PR TITLE
1937 angular implement slide toggle tests

### PR DESCRIPTION
**Pull request by Sudalius**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java  1
- jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/SlideToggleTests.java  1
- jdi-light-angular/src/test/java/io/github/epam/unit/TestsInit.java  1
- jdi-light-angular/src/test/java/io/github/epam/testng/TestNGListener.java  3
- jdi-light-angular/src/main/java/elements/common/SlideToggle.java  2
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java  1
- jdi-light-angular/src/main/java/elements/common/Checkbox.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599256&bid=18343551)

File: jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+public class CheckboxTest {
+
+    private static final String ANGULAR_SELECTED_CLASS = "mat-checkbox-checked";
+    private static final String ANGULAR_DISABLE_CLASS = "mat-checkbox-disabled";
+
+    @InjectMocks
+    private Checkbox checkbox;
+
+    @Mock
+    private UIElement uiElement;
```
Comment (codacy-review) [inline jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java:24]: Codacy Issue found: [Avoid unused private fields such as 'uiElement'.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599256)

File: jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+public class SlideToggleTest {
+
+    private static final String ANGULAR_SELECTED_CLASS = "mat-checked";
+    private static final String ANGULAR_DISABLE_CLASS = "mat-disabled";
+
+    @InjectMocks
+    private SlideToggle slideToggle;
+
+    @Mock
+    private UIElement uiElement;
```
Comment (codacy-review) [inline jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java:26]: Codacy Issue found: [Avoid unused private fields such as 'uiElement'.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599256)

**Comment by mf-code** [comment]
[5442] - 22mOaBn9

**Comment by mf-code** [comment]
[5442] - ox8j34JR

**Comment by mf-code** [comment]
[5442] - YiEgBOtA

**Comment by mf-code** [comment]
[5442] - c1JVlO9i

**Comment by mf-code** [comment]
[5442] - cNiFaY3c

**Comment by mf-code** [comment]
[5442] - dUDWNnho

**Comment by mf-code** [comment]
[5442] - pY7TiHX3

**Comment by mf-code** [comment]
[5442] - Allure report on Netlify: https://5ecf8f46074accf34e7003c8--musing-swartz-6fc060.netlify.app

File: jdi-light-angular/pom.xml [hunk 3]
```diff
@@ -87,7 +108,77 @@
...
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.1</version>
+                <configuration>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>src/test/resources/general.xml</suiteXmlFile>
+                    </suiteXmlFiles>
+                    <testFailureIgnore>true</testFailureIgnore>
+<!--                    <argLine>-->
+<!--                        -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"-->
```
Comment (AkioFaris) [inline jdi-light-angular/pom.xml:127]: Remove this comment

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-angular/src/test/java/io/github/epam/unit/TestsInit.java  1
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java  1
- jdi-light-angular/src/main/java/elements/common/SlideToggle.java  2
- jdi-light-angular/src/test/java/io/github/epam/testng/TestNGListener.java  3
- jdi-light-angular/src/main/java/elements/common/Checkbox.java  2
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java  1
- jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/SlideToggleTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599256&bid=18343551)

**Comment by mf-code** [comment]
[5452] - PqobcujZ

**Comment by mf-code** [comment]
[5452] - A9RoMw4R

**Comment by mf-code** [comment]
[5452] - WRMBZBfz

**Comment by mf-code** [comment]
[5452] - I6M4fuH1

**Comment by mf-code** [comment]
[5452] - NnbQL0bR

**Comment by mf-code** [comment]
[5452] - veYZwWCu

**Comment by mf-code** [comment]
[5452] - 6yv8pDrJ

**Comment by mf-code** [comment]
[5452] - Allure report on Netlify: https://5ed0c9a499dcc3bcff9a85a6--musing-swartz-6fc060.netlify.app

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/SlideToggleTests.java  1
- jdi-light-angular/src/main/java/elements/common/SlideToggle.java  2
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java  1
- jdi-light-angular/src/test/java/io/github/epam/testng/TestNGListener.java  3
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java  1
- jdi-light-angular/src/main/java/elements/common/Checkbox.java  2
- jdi-light-angular/src/test/java/io/github/epam/unit/TestsInit.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599256&bid=18343551)

**Pull request merged**